packages/react-components/src/components/Dialpad/Dialpad.styles.ts

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IStyle, ITextFieldStyles, Theme } from '@fluentui/react';
+
+/**
+ * @private
+ */
+export const containerStyles = (theme: Theme): IStyle => {
+  return {
+    width: '18.75rem',
+    padding: '1.25rem',
+    margin: '1.5rem',
+    textAlign: 'center',
+    background: `${theme.palette.white}`,
+    borderRadius: '0.75rem'
+  };
+};
+
+/**
+ * @private
+ */
+export const rowStyles = (theme: Theme): IStyle => {
+  return {
+    textAlign: 'center',
+    display: 'flex',
+    flexDirection: 'row',
+    flexWrap: 'nowrap',
+    justifyContent: 'space-around',
+    alignItems: 'baseline',
+    background: `${theme.palette.white}`
+  };
+};
+
+/**
+ * @private
+ */
+export const digitStyles = (theme: Theme): IStyle => {
+  return {
+    padding: '0.625rem 1.875rem',
```

Comment:
nit: the current values don't quite match the figma:
| current | figma |
| -- | -- |
| ![image](https://user-images.githubusercontent.com/2684369/167509762-38892141-0c28-491d-8bbf-cf72c02ea089.png) | ![image](https://user-images.githubusercontent.com/2684369/167509789-8040cb07-24df-4822-bdc4-a1f97dde3d95.png) |

Reply:
updated